pkg/util/constants.go

```diff
@@ -52,9 +52,14 @@ const (
 	// FederatedResourceQuotaNameLabel is added to Work to specify associated FederatedResourceQuota's name.
 	FederatedResourceQuotaNameLabel = "federatedresourcequota.karmada.io/name"
 
-	// ManagedByKarmadaLabel is a reserved karmada label to indicate whether resources are managed by karmada controllers.
+	// ManagedByKarmadaLabel is a reserved karmada label to indicate whether resources are member cluster resources
```

Comment:
we'd better keep the first letter capitalized for `Karmada` in comments. others look good to me.
@RainbowMango do you wanna review it again?

Reply:
no big deal.
I'm looking at it.